src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2245,10 +2245,11 @@ public T block(Duration timeout) {
 	 * @param alternate the alternate mono if this mono is empty
 	 *
 	 * @return an alternating {@link Mono} on source onComplete without elements
-	 * @see Flux#switchIfEmpty
+	 * @deprecated Use {@link #switchIfEmpty} instead.
```

Comment:
maybe indicate it will be removed in 3.1.0.RELEASE (or any pre-release after M1)?